GoogleSignIn/Sources/GIDSignIn_Private.h

```diff
@@ -29,12 +29,18 @@ NS_ASSUME_NONNULL_BEGIN
 @class GIDGoogleUser;
 @class GIDSignInInternalOptions;
 @class GTMKeychainStore;
+@class GIDVerifiedAccountDetailResult;
 
 /// Represents a completion block that takes a `GIDSignInResult` on success or an error if the
 /// operation was unsuccessful.
 typedef void (^GIDSignInCompletion)(GIDSignInResult *_Nullable signInResult,
                                     NSError *_Nullable error);
 
+/// Represents a completion block that takes a `GIDVerifiedAccountDetailResult` on success or an
+/// error if the operation was unsuccessful.
+typedef void (^GIDVerifyCompletion)(GIDVerifiedAccountDetailResult *_Nullable verifiedResult,
```

Comment:
I'm not sure this is the right place for this `typedef`. Can you put it on the public header for `GIDVerifyAccountDetail`?

Reply:
Sounds good, moved!